crates/ruff_python_parser/tests/snapshots/invalid_syntax@statements__if_extra_closing_parentheses.py.snap

```diff
@@ -77,4 +76,5 @@ Module(
   |
 2 | if True)):
 3 |     pass
+  |         ^ Syntax Error: Expected a statement
```

Comment:
I only noticed those now. They should probably point at the start of the next line.

Reply:
Yeah this was what I mentioned in one of my other comments. I tried fixing this, but it's hard. I would either need to change the error offsets produced by the parser (which I'm not sure I really want to touch) or I'd need to fix `annotate-snippets` to render empty spans immediately after a line terminator to point at the beginning of the  following line instead of the end of the preceding line.